addons/core/tests/unit/authenticators/password-test.js

```diff
@@ -44,6 +44,7 @@ module('Unit | Authenticator | password', function (hooks) {
     const authMethod = { id: 'paum_1234' };
     this.server.create('scope', { id: 'global', type: 'global' });
     this.server.create('auth-method', { id: authMethod.id, scopeId: 'global' });
+    this.server.create('user', { scopeId: 'global' });
```

Comment:
Is this necessary?

Reply:
Good catch! Forgot about this from my previous mocking of authenticated user. Removed.